src/helper/selection-tools/bounding-box-tool.js

```diff
@@ -105,30 +93,85 @@ class BoundingBoxTool {
             doubleClicked: doubleClicked
         };
         if (this.mode === BoundingBoxModes.MOVE) {
+            this.setCursor(Cursors.GRABBING);
             this._modeMap[this.mode].onMouseDown(hitProperties);
             this.removeBoundsHandles();
         } else if (this.mode === BoundingBoxModes.SCALE) {
             this._modeMap[this.mode].onMouseDown(hitResult, this.boundsPath, getSelectedRootItems());
             this.removeBoundsHandles();
         } else if (this.mode === BoundingBoxModes.ROTATE) {
+            this.setCursor(Cursors.GRABBING);
             this._modeMap[this.mode].onMouseDown(hitResult, this.boundsPath, getSelectedRootItems());
             // While transforming, don't show bounds
             this.removeBoundsPath();
         }
 
         return true;
     }
+    onMouseMove (event, hitOptions) {
+        this._updateCursor(event, hitOptions);
+    }
+    _updateCursor (event, hitOptions) {
+        const {mode, hitResult} = this._determineMode(event, false, hitOptions);
```

Comment:
Let's only call setCursor if the cursor has changed, to avoid spamming actions.

Reply:
Agreed. Fixed in 5542e7e018e08c2ac6db1c75fd1e8d380164232b.